config/Version.public.xcconfig

```diff
@@ -1,2 +1,2 @@
-VERSION_LONG = 25.8.0.2
-VERSION_SHORT = 25.8
+VERSION_LONG = 25.9.0.0
```

Comment:
Did you mean to add these changes to the commit?

Reply:
Needed to update the base, sorry!